recipe/meta.yaml

```diff
@@ -16,9 +16,9 @@
 # about what version you are building.)
 # Examples:
 #       A Public Release for ISIS3.6.1:                        {% set version = "3.6.1" %}
```

Comment:
Should these be updated to match the actual version?

Reply:
I think these are just general examples.